community/vmwh/patches/makefile.patch

```diff
@@ -0,0 +1,13 @@
+--- a/Makefile
++++ b/Makefile
+@@ -23,7 +23,9 @@
+ $(PROG): $(OBJS)
+ 	$(CC) $(OBJS) $(LDPATH) $(LIBS) -o $@
+ 
+-$(OBJS): *.o: *.c
++$(OBJS): vmwh.h
++
++.c.o:
+ 	$(CC) $(CFLAGS) $(INCLUDES) -c $< -o $@
+ 
+ install: all
```

Comment:
Does it refuse to build without this patch?

Reply:
No, it won't build with gnu make.  I did simplify the patch just a bit though.